packages/api/src/controllers/sensorController.js

```diff
@@ -70,6 +70,7 @@ const sensorController = {
       const { access_token } = await IntegratingPartnersModel.getAccessAndRefreshTokens(
         'Ensemble Scientific',
       );
+      const invalidReadingTypes = [];
```

Comment:
@seinnlee putting this here seems to lead to issues if there are more than one sensors with the same error in a single upload. Can you look at how the errors are constructed on line 664 and add them in there, rather than in this top-level call to the parseing function? I believe it should be fairly straightforward to add variables for the errors in there. You would also need to add them everywhere sensors are added to `errorSensors` (probably by spreading the variables from the error into the variables object)